src/checks.rs

```diff
@@ -28,22 +29,34 @@ fn checklist() -> Vec<Box<dyn Check>> {
     ]
 }
 
+fn ending_line_checker() -> Box<dyn Check> {
+    Box::new(ending_blank_line::EndingBlankLineChecker::default())
```

Comment:
Using a `Box` and a separate function for one check is not necessary.

Reply:
I wanted the functions (`ending_line_checker` and `checklist`) to be more unified.
But I will change it, if it is unneccessary.

Reply:
Done